examples/drivenet/drivenet_close_loop.ipynb

```diff
@@ -0,0 +1,329 @@
+    "\n",
+    "## What is a closed-loop evaluation?\n",
+    "In closed-loop evaluation DriveNet is in **full control of the AV**. At each time step, we predict the future trajectory and then move the AV into the first of the DriveNet's predictions. \n",
+    "\n",
+    "At Lyft, we refer to this process with the terms **forward-simulate** or **unroll**.\n",
+    "\n",
+    "![closed-loop](../../images/drivenet/closed-loop.svg)\n",
+    "\n",
+    "This evalaution is crucial to assess the real performance of our DriveNet. **Ideally, we would test the model on the road in the real world**. However, this is clearly very expensive and scales poorly. Forward-simulation is an attempt to evaluate the system in a setting which is closest as possible to a real road test on the same route.\n",
+    "\n",
+    "\n",
+    "## What is a good closed-loop metric?\n",
+    "In this setting **metrics are particularly challenging**. In fact, we would like to penalise some of the simulation drift (e.g. going off road or in the opposite lane) while at the same time allow others (e.g. different speed profiles). As an example, using the same metrics we employed during the open loop evaluation (e.g. ADE) would penalise DriveNet Unconditionally.\n",
+    "\n",
+    "At Lyft L5, we use a large set of different metrics to capture dangerous manoeuvres and behaviours. \n",
+    "\n",
+    "For the sake of simplicity, in this notebook we will be using a very simple proxy to detect if our model is driving in a sensible way, composed of two different metrics:\n",
+    "\n",
+    "### Collisions\n",
+    "Our AV should avoid collisions with other agents. However, not all collisions are created equally. This is because while our AV is fully controlled by DriveNet, other agents are not (a setting we call **log replay**).\n",
+    "\n",
+    "If our AV was slower than the recorded one, chasing agents might bump into us. Clearly, this won't happen in a real setting where other agents can react to our behaviour.\n",
+    "\n",
+    "In this simple example, we won't distinguish between collisions caused by non reactivity of other agents and actual collisions, and we will simply report them all categorised by where they occurred (front, rear and side with respect to the AV).\n",
+    "\n",
+    "However, if we only considered collision, our AV could pass our tests by driving off-road or in a different lane.\n",
+    "\n",
+    "### Distance from Reference Trajectory\n",
+    "To address the issue presented above, we require our DriveNet to loosely stick to the original trajectory in the data. By setting the right threshold on the distance we can allow for different speed profile and small steerings, while pensalising large deviations like driving off-road.\n",
+    "\n",
+    "We can do so by computing the distance between the first predictions from DriveNet and the corresponding annotated positions in world coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Could wee have a heading here or something?
Looks like this cell is about dist. to ref. traj.